src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/Result.css

```diff
@@ -0,0 +1,23 @@
+.result-button {
+    user-select: none;
+
+    overflow: hidden;
+    display: flex !important;
+
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.result-text {
+    display: inline !important;
+    font-size: 0.875rem !important;
+    font-weight: 400 !important;
+}
+
+.before-match,
+.match,
+.after-match {
+    overflow: hidden;
+    flex: 1 1 0; /* Allow each item to take an equal part */
```

Comment:
why?

Reply:
How about this?
.before-match,
.after-match {
    overflow: hidden;
    flex: 1 1 0; /* Allow each item to take an equal part */
    text-overflow: ellipsis;
}

.match {
    overflow: hidden;
    flex: initial;
    text-overflow: ellipsis;
}

It will show results like this
<img width="586" alt="image" src="https://github.com/user-attachments/assets/58d4e4d8-cce9-4a57-b6b2-bc9061e6a813">